frontend/dashboard/pages/PageLayout/PageLayout.test.tsx

```diff
@@ -64,10 +63,45 @@ describe('PageLayout', () => {
 
   test('should redirect to root if user does not have access to selected context', async () => {
     (useParams as jest.Mock).mockReturnValue({
-      selectedContext: 'test',
+      selectedContext: 'testinvalidcontext',
     });
     renderWithMockServices();
     expect(mockedNavigate).toHaveBeenCalledTimes(1);
-    expect(mockedNavigate).toHaveBeenCalledWith(DASHBOARD_ROOT_ROUTE);
+    expect(mockedNavigate).toHaveBeenCalledWith(SelectedContextType.Self, expect.anything());
```

Comment:
Nice! I haven't used `expect.anything()` before, but I understand that it accepts any value except null or undefined, which I perfect match for this case.  Great work! 👏